certified-connectors/DeskDirector/apiProperties.json

```diff
@@ -0,0 +1,58 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "siteHost": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+		  "displayName": "Site Host",
+          "description": "Root site URL (Example: support.deskdirector.com)",
+          "tooltip": "Provide your DeskDirector site URL"
+        }
+      },
+      "apiKey": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+		  "displayName": "API Key",
+          "description": "DeskDirector API Key",
+          "tooltip": "DeskDirector API Key can be retrived from admin portal"
+        }
+      }
+    },
+    "iconBrandColor": "#f2663a",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamicHostUrl",
+        "title": "Set host from connection parameter",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('siteHost')"
+        }
+      },
+      {
+        "templateId": "setheader",
+        "title": "Set authorization header",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "Authorization",
+          "x-ms-apimTemplateParameter.value": "DdApi @connectionParameters('apiKey')",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request"
+        }
+      },
+	  {
+        "templateId": "setconnectionstatustounauthenticated",
+        "title": "Set unauthorized",
+        "parameters": {
+          "x-ms-apimTemplateParameter.statusCode": 401
+        }
+      }
+    ],
+    "publisher": "DeskDirector Limited",
+	  "stackOwner": "DeskDirector Limited"
```

Comment:
@DDAndyChen Indentation -  Everything looks good but the properties in swagger and apiProperties.json are not properly. Can you please fix them to get the PR approved and merged.

Reply:
Cool, I have just fixed the indentation issue.